### PR TITLE
Fix yolk.tanz.xyz

### PR DESCRIPTION
**Pull request by ghost**
https://github.com/AdguardTeam/AdguardFilters/issues/90541

### CODE REVIEW

File: EnglishFilter/sections/adservers.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||ads-partners.coupang.com^$third-party
```
Comment (zloyden) [inline EnglishFilter/sections/adservers.txt:8]: Move it to the `adservers_firstparty.txt` with `$third-party`, if it's necessary.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/adservers.txt (hunk 1 above)
Comment (piquark6046) [reply EnglishFilter/sections/adservers.txt:8]: This domain is globally used by many website.
I think Moving to `adservers_firstparty.txt` is unnecessary.

Comment (zloyden) [reply EnglishFilter/sections/adservers.txt:8]: Well, I don't see a reason that the domain has to be in `adservers.txt`. It's subdomain.

Comment (piquark6046) [reply EnglishFilter/sections/adservers.txt:8]: https://partners.coupang.com/ ([Google Translate](https://translate.google.com/translate?hl=&sl=auto&tl=en&u=https%3A%2F%2Fpartners.coupang.com%2F))
https://partners.coupang.com/#help/tag-usage-guide ([Google Translate](https://translate.google.com/translate?sl=auto&tl=en&u=https://partners.coupang.com/%23help/tag-usage-guide))

Comment (piquark6046) [reply EnglishFilter/sections/adservers.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/90552/commits/ca58117cb25c5b3684b4d8d728c18be9260eaa38

Comment (zloyden) [reply EnglishFilter/sections/adservers.txt]: Ok, but why not `adservers_firstparty.txt`? Subdomains should be there and we usually add them without `$third-party`.

Comment (piquark6046) [reply EnglishFilter/sections/adservers.txt]: Oh, Okay. I'll move it.

Comment (piquark6046) [reply EnglishFilter/sections/adservers.txt]: https://github.com/AdguardTeam/AdguardFilters/pull/90552/commits/76f049ecec88b6a094ff74472d9b3d6621b2ac7a

**Review by zloyden — APPROVED** [review]

**Pull request merged**